### PR TITLE
feat(BUX-669): Removed transportType and changed README

### PR DESCRIPTION
**Pull request by Nazarii-4chain**
# Pull Request Checklist

- [x] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_STANDARDS.md)
- [x] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [x] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [x] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { render } from '@testing-library/react';
-import { App } from './App';
+import { App } from './App.jsx';
```
Comment (chris-4chain) [inline src/App.test.js:2]: Is the `.jsx` part necessary?

Comment (Nazarii-4chain) [reply src/App.test.js:2]: Removed

**Review by chris-4chain — APPROVED** [review]

File: src/pages/login.jsx [hunk 2]
```diff
@@ -114,13 +107,13 @@ const Login = () => {
               }}
             />
           </Box>
-          {!!(config.transportType && config.serverUrl)
+          {!!config.serverUrl
             ?
             (!config.hideServerUrl &&
               <>
```
Comment (arkadiuszos4chain) [inline src/pages/login.jsx:113]: Can we remove empty (`<>`) markups?

**Comment by arkadiuszos4chain** [comment]
I don't understand the scope of the report that this PR is based on.

**Comment by Nazarii-4chain** [comment]
> I don't understand the scope of the ticket that this PR is based on.

Edited Readme after renaming. Added proper links. Removed setTransport because we don't use it anymore. Updated dependnecy

File: src/pages/login.jsx [hunk 3]
```diff
@@ -114,35 +107,20 @@ const Login = () => {
...
           </Box>
-          {!!(config.transportType && config.serverUrl)
+          {!!config.serverUrl
             ?
             (!config.hideServerUrl &&
-              <>
                 <TextField
                   fullWidth
-                  label={config.transportType}
+                  label={"http"}
```
Comment (arkadiuszos4chain) [inline src/pages/login.jsx:115]: remove '{' and '}'

**Comment by arkadiuszos4chain** [comment]
Why did you add empty `pages/admin-login.jsx`?

**Review by arkadiuszos4chain — CHANGES_REQUESTED** [review]

**Comment by Nazarii-4chain** [comment]
> Why did you add empty `pages/admin-login.jsx`?

Removed

File: src/pages/login.jsx (hunk 3 above)
Comment (Nazarii-4chain) [reply src/pages/login.jsx]: Removed

**Review by arkadiuszos4chain — APPROVED** [review]

**Review by chris-4chain — APPROVED** [review]

**Pull request merged**